### PR TITLE
improvement(results_latency_during_ops.html): add legend to table result

### PR DESCRIPTION
**Pull request by fgelcer**
the tables have results of latency, and to be sure
every reader is in the same page, this commit will
be adding a legend to the tables, that the latency
results are displayed in ms (miliseconds).

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/results_analyze/__init__.py [hunk 1]
```diff
@@ -188,6 +188,7 @@ def check_regression(self, test_id, data, is_gce=False):  # pylint: disable=too-
         last_events, events_summary = self.get_events(event_severity=[
             Severity.CRITICAL, Severity.ERROR, Severity.WARNING])
         reactor_stall_events, reactor_stall_events_summary = self.get_debug_events()
+        self.log.info(f'FFFFFF reactor_stall_events_summary={reactor_stall_events_summary} FFFFFF')
```
Comment (fgelcer) [inline sdcm/results_analyze/__init__.py:191]: here i need to remove this debugging message, and in the previous line, i might want to filter out only the reactor stall events (from within debug events)

**Comment by fgelcer** [comment]
giving it a ride in [this build](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/fix_collect_latency_issues/job/perf-latency-regression-collect-latency-fix/8/)

**Comment by fgelcer** [comment]
@aleksbykov , this is ready for review

**Review by aleksbykov — APPROVED** [review]
LGTM, Could you attach screenshot with report?

**Comment by fgelcer** [comment]
> LGTM, Could you attach screenshot with report?

here they are (both commits, with both new information):

Screenshot from 2021-05-29 20-57-54

**Review by roydahan — APPROVED** [review]

**Comment by fgelcer** [comment]
the back port was partial and 1 commit is still missing to be back ported, FYI

**Pull request merged**